documentation/asciidoc/topics/ref_crd.adoc

```diff
@@ -16,7 +16,7 @@ include::yaml_annotated/cr_minimal.yaml[]
 <1> declares the version.
 <2> sets the resource type that the {ispn_operator} uses to handle the lifecycle of {brandname} clusters.
 <3> names the {brandname} cluster.
-<4> sets the number of pods in the {brandname} cluster.
+<4> sets the number of pods in the {brandname} cluster. If you set the value to `0` the {ispn_operator} stops the running cluster.
```

Comment:
see comment above for the restart procedure

Reply:
I think it's better to just keep this simple. better to say it sets the number of pods. we can talk about graceful shutdown in its own topic.